server/controllers/dietaryRequirementsController.ts

```diff
@@ -71,7 +70,11 @@ export default class DietaryRequirementsController {
       }
 
       // Remove page as this comes from the API
-      const resp = await this.dietReportingService.getDietaryRequirementsForPrison(clientToken, prisonId, queryParams)
+      const resp = await this.dietReportingService.getDietaryRequirementsForPrison(
+        res.locals.user.token,
```

Comment:
Why are we switching to use the user token here? The general advice is to use the client token where possible.